main.css

```diff
@@ -41,4 +40,98 @@ li {
   width: 20px;
   height: 20px;
   margin-right: 15px;
+}
+
+/* Popup container */
```

Comment:
nu prea vad rostul comentariilor.
clasele de css ar trebui sa fie evident denumite cat sa iti dai seama ce e (si sunt)
un exemplu cand ar fi util un comentariu in css, e cand ai un fix pt un browser anume